### PR TITLE
feat: workflow to stable

### PR DESCRIPTION
**Pull request by mikeee**
# Description

Changes to mark the workflow API as stable

## Issue reference

https://github.com/dapr/dapr/issues/7156 [tentative - phase 2 planning?]

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: pkg/grpc/universalapi/api_workflow.go [hunk 1]
```diff
@@ -269,6 +269,41 @@ func (a *UniversalAPI) PurgeWorkflowBeta1(ctx context.Context, in *runtimev1pb.P
...
+	return a.ResumeWorkflow(ctx, in)
+}
+
+// PurgeWorkflowBeta1 is the API handler for purging a workflow
+func (a *UniversalAPI) PurgeWorkflowBeta1(ctx context.Context, in *runtimev1pb.PurgeWorkflowRequest) (*emptypb.Empty, error) {
+	return a.PurgeWorkflow(ctx, in)
+}
+
 // GetWorkflowAlpha1 is the API handler for getting workflow details
 func (a *UniversalAPI) GetWorkflowAlpha1(ctx context.Context, in *runtimev1pb.GetWorkflowRequest) (*runtimev1pb.GetWorkflowResponse, error) {
```
Comment (ItalyPaleAle) [inline pkg/grpc/universalapi/api_workflow.go:308]: Should Alpha1 methods be removed in this PR? They are aliases for now

Comment (mikeee) [reply pkg/grpc/universalapi/api_workflow.go:308]: I'll take advice on this from anyone - not sure of the timelines etc for deprecating this.

Comment (ItalyPaleAle) [reply pkg/grpc/universalapi/api_workflow.go:308]: Not sure either, but wanted to bring that up

**Comment by mikeee** [comment]
Closing this PR. This is a relatively easy change to make for the coming releases.

**Pull request closed without merging**